tests/aws/services/kms/test_kms.snapshot.json

```diff
@@ -2286,5 +2286,256 @@
         }
       }
     }
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_list_key_rotations_after_on_demand_rotation": {
+    "recorded-date": "15-07-2025, 15:09:01",
+    "recorded-content": {
+      "initial-rotations": {
+        "Rotations": [],
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      },
+      "after-on-demand-rotation": {
+        "Rotations": [
+          {
+            "KeyId": "<key-id:1>",
+            "KeyMaterialId": "key-material-id",
+            "KeyMaterialState": "CURRENT",
+            "RotationDate": "datetime",
+            "RotationType": "ON_DEMAND"
+          }
+        ],
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      }
+    }
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_list_key_rotations_different_key_states": {
+    "recorded-date": "16-07-2025, 15:50:18",
+    "recorded-content": {
+      "enabled-key-rotations": {
+        "Rotations": [
+          {
+            "KeyId": "<key-id:1>",
+            "KeyMaterialId": "key-material-id",
+            "KeyMaterialState": "CURRENT"
+          }
+        ],
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      },
+      "disabled-key-rotations": {
+        "Rotations": [
+          {
+            "KeyId": "<key-id:1>",
+            "KeyMaterialId": "key-material-id",
+            "KeyMaterialState": "CURRENT"
+          }
+        ],
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      },
+      "pending-deletion-key-rotations": {
+        "Rotations": [
+          {
+            "KeyId": "<key-id:1>",
+            "KeyMaterialId": "key-material-id",
+            "KeyMaterialState": "CURRENT"
+          }
+        ],
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      }
+    }
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_list_key_rotations_unsupported_key_types[RSA_2048-SIGN_VERIFY-UnsupportedOperationException]": {
+    "recorded-date": "16-07-2025, 15:59:30",
+    "recorded-content": {}
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_list_key_rotations_unsupported_key_types[RSA_4096-ENCRYPT_DECRYPT-UnsupportedOperationException]": {
+    "recorded-date": "16-07-2025, 15:59:31",
+    "recorded-content": {}
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_list_key_rotations_unsupported_key_types[ECC_NIST_P256-SIGN_VERIFY-UnsupportedOperationException]": {
+    "recorded-date": "16-07-2025, 15:59:31",
+    "recorded-content": {}
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_list_key_rotations_unsupported_key_types[HMAC_256-GENERATE_VERIFY_MAC-UnsupportedOperationException]": {
+    "recorded-date": "16-07-2025, 15:59:32",
+    "recorded-content": {}
+  },
```

Comment:
Are these the leftovers? Because i cant see related tests for `UnsupportedOperationException` specifically?